### ISSUE TITLE
Can't navigate editor tabs using keyboard

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/exercism.io#6059]
TL;DR: According to [W3 AI Aria Practice TabsList Example](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html), at least `Left Arrow`, `Right Arrow`, `Home` and `End` should work.

---

Currently, there are (at least) two places in the editor where I don't understand how to navigate without using a pointing device.

![File tab list in the Exercism Editor](https://user-images.githubusercontent.com/1964376/138499493-b576c0c4-d528-463f-a09e-7a04fec45269.png)

![Tabs tab list in the Exercism Editor](https://user-images.githubusercontent.com/1964376/138499540-98209bf8-e66a-49b0-978d-bfd046e6fd27.png)

 Both have the role `tablist` with each button being a role `tab` (which is great), but neither responds to the keyboard inputs. Because the `tab` buttons (correctly) have `tabindex=-1` (making the list have a single tab-stop, which is correct), you can't workaround this by _tabbing_. How ironic.

**Comment by avidrucker** [comment]
+1 On this issue from me as well. As a learner, I currently cannot navigate/switch/toggle between the "Instructions tab" and the "Results tab" using the Tab key (or arrow keys) in Accessibility Mode.

Browser: I am using Chrome Version 96.0.4664.45 (Official Build) (64-bit) on Linux Mint.

![image](https://user-images.githubusercontent.com/6962664/150645502-87e1be07-1584-4ced-a983-5c5ee6d32c18.png)
